presto-tpcds/src/test/java/com/facebook/presto/tpcds/TestTpcdsMetadataStatistics.java

```diff
@@ -139,8 +122,7 @@ public void testTableStatsDetails()
                 ColumnStatistics.builder()
                         .setNullsFraction(Estimate.of(0))
                         .setDistinctValuesCount(Estimate.of(1))
-                        .setLowValue(Optional.of(-500L))
-                        .setHighValue(Optional.of(-500L))
+                        .setRange(new DoubleRange(-5, -5))
```

Comment:
Shouldn't this be `new DoubleRange(-500, -500)`?

Reply:
No. The actual value is `5`. But the internal representation is `500`, because the decimal type has the scale set to `2`